modules/howtos/pages/provisioning-cluster-resources.adoc

```diff
@@ -98,8 +98,51 @@ The `Flush` operation may fail if the bucket does not have flush enabled, in tha
 
 == Collection Management
 
-This is a xref:6.5@server:developer-preview:collections/collections-overview.adoc[Developer Preview feature], see the https://docs.couchbase.com/sdk-api/couchbase-java-client/com/couchbase/client/java/manager/collection/CollectionManager.html[API doc].
+The CollectionManager interface may be used to create and delete scopes and collections from the Couchbase cluster.
+It is instantiated through the `Bucket.collections()` method.
+Refer to the https://docs.couchbase.com/sdk-api/couchbase-java-client/com/couchbase/client/java/manager/collection/CollectionManager.html[`CollectionManager` API documentation] --
+and to its https://docs.couchbase.com/sdk-api/couchbase-java-client/com/couchbase/client/java/manager/collection/AsyncCollectionManager.html[Async counterpart] --
+for further details.
+
+[source,java]
+----
+include::example$CollectionManagerExample.java[tag=create-collection-manager,indent=0]
+----
+
+You can create a scope:
+
+[source,java]
+----
+include::example$CollectionManagerExample.java[tag=create-scope,indent=0]
+----
+
+You can then create a collection within that scope:
+
+[source,java]
+----
+include::example$CollectionManagerExample.java[tag=create-collection,indent=0]
+----
+
+Finally, you can drop unneeded collections and scopes:
+
+[source,java]
+----
+include::example$CollectionManagerExample.java[tag=drop-collection,indent=0]
+
+include::example$CollectionManagerExample.java[tag=drop-scope,indent=0]
+----
 
+Note that you require at least the https://docs.couchbase.com/server/7.0/learn/security/roles.html#bucket-admin[Bucket Admin] privilege to create or drop a Scope.
+The minimum privilege for managing Collections is https://docs.couchbase.com/server/7.0/learn/security/roles.html#manage-collections-in-scope[Manage Collections in Scope].
```

Comment:
User also needs "Data Reader" (or something similar) or else they can't open the bucket to get the collections manager.

Reply:
Thanks! In fact I'd already added data reader in the *code*, but wasn't sure that was the best practice, will document.